speechbrain/dataio/dataio.py

```diff
@@ -390,7 +405,9 @@ def read_audio_multichannel(waveforms_obj, backend=None):
     """
     if backend not in [None, "ffmpeg", "sox", "soundfile"]:
         raise ValueError(
-            "backend must be one of 'ffmpeg', 'sox', 'soundfile' or None"
+            "backend must be one of 'ffmpeg', 'sox', 'soundfile' or None",
+            "Available backends on your system: ",
+            torchaudio.list_audio_backends(),
         )
```

Comment:
Not a huge deal but this code seems repeated from above, any way to combine (e.g. unified audio backend checker function)